scripts/migration/dataset_paths_ecs_rollback.py

```diff
@@ -54,12 +54,15 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument('-f', '--fields-to-map', dest='fieldstomap', choices=[MAPPING_FIELD_HDFS_PATH, MAPPING_FIELD_HDFS_PUBLISH_PATH, MAPPING_FIELD_HDFS_ALL],
                         default=MAPPING_FIELD_HDFS_ALL, help="Rollback either item's 'hdfsPath', 'hdfsPublishPath' or 'all'.")
 
-    parser.add_argument('-d', '--datasets', dest='datasets', metavar="DATASET_NAME", default=[],
-                        nargs="+", help='list datasets names to rollback path changes in (hint: MTs are also datasets).')
-
     parser.add_argument('-o', '--only-datasets', dest='onlydatasets', action='store_true', default=DEFAULT_DATASETS_ONLY,
```

Comment:
I am not sure if this option is needed when introducing -m and there is logic for detection of both -m and -d usage.

Reply:
It is a different usecase:
`-d` remaps datasets and possibly MTs **by the dataset name** (if `-o` is specified, only datasets)
`-m` remaps MTs **by MT name**